### PR TITLE
Add bicycle_road=yes

### PR DESCRIPTION
**Pull request by ad47**
http://wiki.openstreetmap.org/wiki/Key:bicycle road
Bicycle roads are much better for cyclists than common roads, so it worth to prefer them in routing.xml and render additional dashed line similar to cycleway=lane, but using cycleway style.
https://www.dropbox.com/s/bh9tk47touiz6e9/bicycle_road.png?dl=0

### CODE REVIEW

File: rendering_styles/default.render.xml [hunk 1]
```diff
@@ -2181,6 +2181,11 @@
 					<apply_if appMode="default" moreDetailed="false" maxzoom="15" order="-1"/>
 				</apply_if>
 			</case>
+			<case appMode="bicycle" tag="bicycle_road" value="yes" order="143">
+				<apply_if layer="1" order="144">
+					<apply_if appMode="default" moreDetailed="false" maxzoom="15" order="-1"/>
```
Comment (xmd5a2) [inline rendering_styles/default.render.xml]: This condition will never happen because it is inside of appMode="bicycle"

**Comment by ad47** [comment]
Sounds logical, so I removed both redundant lines.

**Comment by aceman444** [comment]
What is this patch about? I see no bicycle_road key on that linked wiki page.
Also that key is never used in the database: http://taginfo.openstreetmap.org/search?q=bicycle_way#keys 

There must be some mistake here. Please revert this until it is resolved.

**Comment by xmd5a2** [comment]
Bicycle "road", not "way". There is a space in a link.

**Comment by aceman444** [comment]
I see, the link went to Key:bicycle ;)

**Pull request merged**